### PR TITLE
Add autoyast profiles to test multipath

### PR DESCRIPTION
**Pull request by rwx788**
We got profile from one of our beta customers which we have adjusted for
sle 12 sp3 and sle 15. Profile testing profile requires multipath to be
configured. In openQA we have special worker class for this. Hence, no
local verification run. Tested manually. On SLE15 installation fails due to reported bug, minor adjustments to the software section may be required.

Profile tests multipath and dynamic changes to the profile using pre-scripts feature.

Test suite requires MULTIPATH setting set to true, and AUTOYAST setting pointing to the profile.

- Related ticket: [poo#20818](https://progress.opensuse.org/issues/20818).

### CODE REVIEW

File: data/autoyast_sle12/autoyast_multipath.xml [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+    <pre-scripts config:type="list">
+      <script>
+        <debug config:type="boolean">true</debug>
+        <filename>pre_disklayout</filename>
+        <interpreter>shell</interpreter>
+        <source><![CDATA[
+if [ ! -e /tmp/profile/modified.xml ]; then
+        cp /tmp/profile/autoinst.xml /tmp/profile/modified.xml
+fi
+# Enable mutlipath
```
Comment (okurz) [inline data/autoyast_sle12/autoyast_multipath.xml]: so who entered the typo here, you or *the customer*? ;)

Comment (rwx788) [reply data/autoyast_sle12/autoyast_multipath.xml]: Hehe, it's our beta customer. I'll add a comment with description to the profile and test suite description

**Comment by okurz** [comment]
@asmorodskyi did you overlook or overrule my comment?

File: data/autoyast_sle12/autoyast_multipath.xml (hunk 1 above)
Comment (rwx788) [reply data/autoyast_sle12/autoyast_multipath.xml]: @okurz Please, review: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3999

**Comment by asmorodskyi** [comment]
both in some extend. I have seen it but your second "informal" part lead me to conclusion that you don't have any real issues with this PR but just thinking loud about "bright future".

File: data/autoyast_sle12/autoyast_multipath.xml (hunk 1 above)
Comment (asmorodskyi) [reply data/autoyast_sle12/autoyast_multipath.xml]: it's crazy , I have read this line 10 times and only after @rwx788  provide another PR I noticed that in this version it was "muTLipath" :\ . Before I was question myself "Where @okurz  see error here" ))))))))))))))))

**Pull request merged**